### PR TITLE
Changed location configuration value to fix #1

### LINKED ISSUE #1: tinyproxy.conf location to override is /etc/tinyproxy.conf
It was ignoring my config file until I realised this!  Thanks.

### PR DESCRIPTION
**Pull request by cokeSchlumpf**
Like mentioned in issue #1 - the config file location is not correct with currrent versions of tinyproxy (and its actually time consuming to find out) ... please have a look and merge if possible :)

### CODE REVIEW

File: tinyproxy.conf [hunk 1]
```diff
@@ -207,7 +207,7 @@ MaxRequestsPerChild 0
 # The order of the controls are important. All incoming connections are
 # tested against the controls based on order.
 #
-Allow 127.0.0.1
```
Comment (JubbaSmail) [inline tinyproxy.conf]: thx for your PR,
however, why did you remove Allow 127.0.0.1 ?

Comment (cokeSchlumpf) [reply tinyproxy.conf]: It was just more convenient when you use the image for testing with Docker: Basically it allows all clients to use the proxy - as other containers as well as the host machine will have another client ip address than `127.0.0.1` to the running proxy.

Comment (cokeSchlumpf) [reply tinyproxy.conf]: ... but of course we can change it back if you require the default setting in your setup.

Comment (JubbaSmail) [reply tinyproxy.conf]: I see, okay then
